types/tpdirect/README.md

```diff
@@ -0,0 +1,5 @@
+# TPDirect Usage Note
+
+This currenct version match the [TapPay Docs SDK v5.5](https://docs.tappaysdk.com/tutorial/en/home.html).  
+TapPay doesn't prodive the npm installation, but provide the global variable `TPDirect`. 
```

Comment:
I think it's no need for now.  
I add README for new version, it could be a comparison for new SDK version, like this.

1.0 -- 5.5.0
1.1 -- 5.6.0
2.0 -- 6.0.0

But, if I could add the SDK version in the header comment of index.d.ts, I think I could remove this file.

Reply:
I removed this.